### ISSUE TITLE
License

### ISSUE THREAD

**Issue by sclarke81** [XRobots/BlinkyBall#1]
Hi James,

I think it's great that you're sharing your work on GitHub. You should add a license file though so it's clear what people can (or can't!) do with it.

Keep up the great work!

Kind regards,

Stephen

**Comment by XRobots** [comment]
Yes I thought about that, however there's already a license in Jeff Rowberg's MPU6050 code which is in the code, so I can't really re-license that code. Not sure what's an appropriate license in this case.

**Comment by sclarke81** [comment]
I'd suggest applying MIT (assuming you're happy with that for your contributions). You've rightly kept his copyright notice so you'd just be explicitly stating your intent for for your work (and in a way that doesn't contravene Jeff's license). As well as adding a license file you could add a license section to your readme to explain the situation.

Note that I'm not a legal expert in any way, but just have some familiarity with licensing from my work as a software engineer.

You could also refactor your code to keep Jeff's code separate and just include it (it'd make it easier to update as well). Then you'd easily keep your licensing separate.

I'd be happy to raise a pull request with the licensing suggestions if that'd help?

**Comment by XRobots** [comment]
Thanks, sounds good to me!
